indico/modules/events/abstracts/__init__.py

```diff
@@ -134,6 +134,11 @@ def _reviewing_area_visible(conf):
             return False
         return bool(get_user_tracks(conf.as_event, session.user))
 
+    def _submit_abstract_visible(conf):
+        if not session.user or not conf.has_feature('abstracts'):
```

Comment:
Then I wouldn't hide the menu item at all if the submission period is active or in the future. It's just confusing if you see something while not logged in but not when logged in.